source/plasma3DClass.py

```diff
@@ -508,31 +522,69 @@ def __init__(self):
 		self.psimin = None
 		self.Lc = None		# in km
 		self.N = 1
-		self.NCPUs = 100
 		self.q = np.zeros(self.N)
 		self.q0 = None
 		self.ep = None	# equilParams_class instance for EFIT equilibrium
 		self.HFS = None	# True: use high field side SOL, False: use low field side SOL
+		
+		#Default inputs
+		self.NCPUs = 100
 		self.teProfileData = None
 		self.neProfileData = None
+		
+    
+	def allowed_class_vars(self):
+		"""
```

Comment:
@wingena same thing here.  please add a markdown compatible description for each variable here.  Additionally, you will need to add the following line to  docs/source/inputFile.rst:
```
.. autofunction:: plasma3DClass.heatflux3D.allowed_class_vars  
```
in order to get your comments added to the readthedocs.  I only did the plasma3D class in my commit, so you will need to add the line above to the inputFile.rst on your own (a good exercise to understand how it works).

Reply:
Same here, I updated it.